wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Insights > All Time Stats

Fixes #10466 

To test:

---
On a site that has stats for Posts, Views, Visitors, and Best Views Ever, verify all four appear in All Time Stats:

<img width="350" alt="all" src="https://user-images.githubusercontent.com/1816888/48747069-3d3fa780-ec2f-11e8-9e20-19991f48da9a.png">

---
On a site that does _not_ have all four stats, verify only those that do appear in All Time Stats:

<img width="350" alt="partial" src="https://user-images.githubusercontent.com/1816888/48747139-8263d980-ec2f-11e8-9d5a-bd9c897e27af.png">

---
On a site that has no All Time Stats, verify the `No data yet` view is displayed:

<img width="350" alt="none" src="https://user-images.githubusercontent.com/1816888/48747205-c060fd80-ec2f-11e8-8c22-eee70a311b5c.png">
